FNPlugin/Propulsion/ThermalNozzleController.cs

```diff
@@ -1638,6 +1638,10 @@ public double GetNozzleFlowRate()
         public void EstimateEditorPerformance()
         {
             var atmospherecurve = new FloatCurve();
+            if (myAttachedEngine == null)
+            {
+                return;
+            }
```

Comment:
Is it intentional to change module behaviour in the same PR as in-game description text?

Reply:
Yes, it was intended, although probably not the greatest style, since it was only a bug fix for an NRE thrown on vessel pack. If it is better to make a separate PR I can split it up in the future.